src/main/kotlin/no/nav/familie/ba/sak/behandling/resultat/BehandlingsresultatUtils.kt

```diff
@@ -189,16 +189,19 @@ object BehandlingsresultatUtils {
                         // Håndtering av teknisk opphør.
                         TIDENES_MORGEN.toYearMonth()
                     } else if (resultater.contains(YtelsePersonResultat.OPPHØRT) &&
-                               segmenterLagtTil.isEmpty && segmenterFjernet.size() == 1) {
+                               segmenterLagtTil.isEmpty && segmenterFjernet.size() > 0) {
```

Comment:
```suggestion
                               segmenterLagtTil.isEmpty && segmenterFjernet.isNotEmpty()) {
```

Er dette kanskje mer lesbart?

Reply:
Ja, selvfølgelig. Skjønner ikke hvorfor jeg plutselig skulle finne på å gjøre det sånn. Thx!